<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AJTrackingFrame</class>
 <widget class="QDialog" name="AJTrackingFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSlider" name="frameSlider">
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="renderingWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="beforeGroupBox">
            <property name="title">
             <string>t</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QVTKWidget" name="beforeQVTKWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Select</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="selectVertexBeforeButton">
                  <property name="text">
                   <string>vertex</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="selectEdgeBeforeButton">
                  <property name="text">
                   <string>edge</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="selectCellBeforeButton">
                  <property name="text">
                   <string>cell</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="beforeSelectLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="afterGroupBox">
            <property name="title">
             <string>t</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QVTKWidget" name="afterQVTKWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Select</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="selectVertexAfterButton">
                  <property name="text">
                   <string>vertex</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="selectEdgeAfterButton">
                  <property name="text">
                   <string>edge</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="selectCellAfterButton">
                  <property name="text">
                   <string>cell</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="afterSelectLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addCorrespondenceButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="computeEstimationsButton">
         <property name="text">
          <string>Guess Correspondences</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabsWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QGroupBox" name="succesorsGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Correspondences</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QPushButton" name="deleteCorrespondenceButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QTableWidget" name="correspondencesTableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSelectVertexBefore">
   <property name="text">
    <string>Select Vertex Before</string>
   </property>
  </action>
  <action name="actionSelectEdgeBefore">
   <property name="text">
    <string>Select Edge Before</string>
   </property>
  </action>
  <action name="actionSelectCellBefore">
   <property name="text">
    <string>Select Cell Before</string>
   </property>
  </action>
  <action name="actionSelectVertexAfter">
   <property name="text">
    <string>Select Vertex After</string>
   </property>
  </action>
  <action name="actionSelectEdgeAfter">
   <property name="text">
    <string>Select Edge After</string>
   </property>
  </action>
  <action name="actionSelectCellAfter">
   <property name="text">
    <string>Select Cell After</string>
   </property>
  </action>
  <action name="actionGuessCorrespondences">
   <property name="text">
    <string>Guess Correspondences</string>
   </property>
  </action>
  <action name="actionDeleteCorrespondence">
   <property name="text">
    <string>Delete Correspondence</string>
   </property>
  </action>
  <action name="actionAddCorrespondence">
   <property name="text">
    <string>Add Correspondence</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>selectCellAfterButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectCellAfter</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectCellBeforeButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectCellBefore</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectEdgeAfterButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectEdgeAfter</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectEdgeBeforeButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectEdgeBefore</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectVertexAfterButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectVertexAfter</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectVertexBeforeButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectVertexBefore</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addCorrespondenceButton</sender>
   <signal>clicked()</signal>
   <receiver>actionAddCorrespondence</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteCorrespondenceButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDeleteCorrespondence</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>900</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
